lol/misc/maps.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Maps data"
+authors: "@Canisback" 
+tags: "maps,map" # A comma separated list of the tags this document has
+markers: "maps,map" # A comma separated list of all the markers this document has
```

Comment:
I know it's not a well documented feature, but can you change markers to be just "generic"?
Ready to merge it in once you make the change